src/Common/StringUtils.h

```diff
@@ -25,4 +26,6 @@ namespace openblack::string_utils
 
 [[nodiscard]] std::string Capitalise(const std::string& str);
 
+[[nodiscard]] std::vector<std::string> Split(const std::string& string, const std::string& delimiter);
```

Comment:
Could you look for more places where we split strings in the code and refactor them with this.

Reply:
Couldn't find anything. Any ideas?